crates/l3/appchain-core-contract-client/src/interfaces/messaging.rs

```diff
@@ -0,0 +1,90 @@
+use common::invoke_contract;
+use common::LocalWalletSignerMiddleware;
+use starknet_accounts::Execution;
+use starknet_core::types::FieldElement;
+use std::sync::Arc;
+use color_eyre::Result;
+
+pub struct Messaging {
+    client: Arc<LocalWalletSignerMiddleware>,
+    address: FieldElement,
+}
+
+impl Messaging {
+    pub fn new(address: FieldElement, client: Arc<LocalWalletSignerMiddleware>) -> Self {
+        Self { client, address }
+    }
+
+    pub async fn send_message_to_appchain(
+        &self,
+        to_address: FieldElement,
+        selector: FieldElement,
+        payload: Vec<FieldElement>,
+        // ) -> Result<Option<Execution<LocalWalletSignerMiddleware>>, Error> {
```

Comment:
let's remove this comment, and any other code comments we've in code

Reply:
done